stripe/src/main/java/com/stripe/android/model/CardBrand.kt

```diff
@@ -66,14 +66,20 @@ enum class CardBrand(
         cvcLength = setOf(3, 4),
         defaultMaxLength = 15,
         pattern = Pattern.compile("^(34|37)[0-9]*$"),
+        partialPatterns = mapOf(
+            1 to Pattern.compile("^3$")
+        ),
```

Comment:
Should we update `partialPatterns` to not have a default value in the constructor?